controllers/nstemplatetier/nstemplatetier_controller_test.go

```diff
@@ -56,28 +52,20 @@ func TestReconcile(t *testing.T) {
 			tiertest.AssertThatNSTemplateTier(t, "basic", cl).
 				HasStatusUpdatesItems(1).
 				HasLatestUpdate(toolchainv1alpha1.NSTemplateTierHistory{
-					Hash:           basicTier.Labels["toolchain.dev.openshift.com/basic-tier-hash"],
-					Failures:       0,
-					FailedAccounts: []string{},
+					Hash: basicTier.Labels["toolchain.dev.openshift.com/basic-tier-hash"],
 				})
-			turtest.AssertThatTemplateUpdateRequests(t, cl).TotalCount(0) // no TemplateUpdateRequest created yet
 		})
 
-		t.Run("with previous complete entries", func(t *testing.T) {
+		t.Run("with previous entries", func(t *testing.T) {
 			// given
-			now := metav1.Now()
 			basicTier := tiertest.BasicTier(t, tiertest.CurrentBasicTemplates, tiertest.WithPreviousUpdates(
 				toolchainv1alpha1.NSTemplateTierHistory{
-					StartTime:      metav1.Now(),
-					CompletionTime: &now,
-					Failures:       0,
-					FailedAccounts: []string{},
+					StartTime: metav1.Now(),
+					Hash:      "abc123",
 				},
 				toolchainv1alpha1.NSTemplateTierHistory{
-					StartTime:      metav1.Now(),
-					CompletionTime: &now,
-					Failures:       2,
-					FailedAccounts: []string{"failed1", "failed2"},
+					StartTime: metav1.Now(),
+					Hash:      "def456",
 				}))
 			initObjs := []runtime.Object{basicTier}
 			r, req, cl := prepareReconcile(t, basicTier.Name, initObjs...)
```

Comment:
take a look at the content of the `prepareReconcile` function - this can be simplified (shortened) as well

Reply:
Good catch, done in 0ce848e2b13fe954c116bdf3f4945890a3a6803a